features/api-token.feature

```diff
@@ -13,14 +13,16 @@ Feature: User API Token
         - Users can revoke a token at any time
         - No one can ever look up the raw value after initially generating the token
 
+
     Background:
-        Given the user "calvin" exists
+        Given "calvin" is logged in
 
-    @ignore
     Scenario: Generate New API Token
-        When a new API token is generated
+        And does not own a token named "tiger"
```

Comment:
The `Background` directive is new to me, but it feels really weird in any case that Scenarios aren't using a `Given-When-Then` style.

I'd change this to `Given "calvin" does not own a token named "tiger"`